verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -149,7 +152,7 @@ export default class CasesController {
             );
             const result = await users().findOneAndUpdate(
                 {
-                    _id: new ObjectId(user.id),
+                    _id: new ObjectId(user._id),
```

Comment:
I believe that `_id` is returned from MongoDB, I just made this change to be consistent. I think I saw some typescript errors due to using `id` instead of `_id`. I will check that